Sources/AsyncHTTPClient/HTTPClient.swift

```diff
@@ -403,6 +417,14 @@ public class HTTPClient {
             return EventLoopPreference(.delegateAndChannel(on: eventLoop))
         }
     }
+
+    /// Specifies decompression settings.
+    public enum Decompression {
+        /// Decompression is disabled.
+        case disabled
+        /// Decompression is enabled.
+        case enabled(limit: NIOHTTPDecompression.DecompressionLimit)
```

Comment:
This means that users are required to `import NIOHTTPCompression` in order to use decompression. Are people OK with that?

You can avoid this by doing:

```swift
public typealias DecompressionLimit = NIOHTTPCompression.DecompressionLimit
```

Reply:
I don't think they are, one can just do `enabled(limit: .size(10))`, no?

Reply:
Oh you're right, cool! Sorry about that.